historical_system_profiles/db_interface.py

```diff
@@ -39,7 +42,10 @@ def get_hsps_by_inventory_id(inventory_id, account_number):
     )
 
     query_results = query.all()
-    return query_results
+
+    if query_results:
+        return query_results
+    raise HTTPError(HTTPStatus.NOT_FOUND, message="%s not available" % inventory_id)
```

Comment:
this error should be "no historical profiles found for inventory_id %s"

Reply:
crap i hit merge before this got fixed, sorry :(

If you don't mind, can you put in a new PR to update the message?